daemon/daemon.go

```diff
@@ -455,20 +455,24 @@ func policyCommitMessage(us policy.Updates, cause update.Cause) string {
 func policyEventTypes(u policy.Update) []string {
 	types := map[string]struct{}{}
```

Comment:
This is a common Go pattern used to simulate a Set. Without this we'd end up with duplicates.